CIP-0116/README.md

```diff
@@ -247,21 +242,57 @@ Instead of providing all three variants of encoding, we base the schema on the o
+
+Furthermore, for each subsequent schema, the [changelog](./changelog.md) must be updated. Authors must clearly articulate the deltas between schemas.
+
 ## Rationale: how does this CIP achieve its goals?
 
-## Path to Active
+### Scope
 
-- [ ] Complete the specification
-- [ ] Provide an implementation of validating functions that uses this json-schema
-- [ ] Collect a list of cardano domain types implementations and negotiate transition to the specified formats with maintainers (if it makes sense and is possible)
+We keep a lean scope, limited to the data types within Cardano blocks. The rationale for this is that block data is by far the most useful for the majority of Cardano actors. There is also one nice benefit that the definitions can map directly from the provided CDDL file from ledger team.
+
+Defining additional would likely add unneeded overhead to the maintenance of this standard.
+
+### Strictness
+
+This CIP lays out strong conventions that future schema authors must follow. With a large set of design principles and conventions. The aim for these is to minimize the potential for unavoidable deltas between schemas.
+
+By setting sometimes arbitrary conventions we hope to create a single possible interpretation from CBOR to JSON. This is beneficial as it allows the development of downstream standards, which rely on a canonical JSON representation.
+
+### Absence of extensibility
+
+The schemas MUST NOT be extensible with additional properties. This may sound counter-intuitive and against the spirit of json-schema, but there are some motivations behind that:
+
+- More safety from typos: object fields that are optional may be specified with slightly incorrect names in dApps' code, leading to inability of the decoders to pick up the values, which may go unnoticed.
+- Clear delineation between Cardano domain types and user dApp domain types: forcing the developers to store their dApp domain data separately from Cardano data, or close to it (as opposed to mixing these together in a single object) will indirectly motivate better structured dApp code.
+
+### JSON
+
+JSON was chosen as there is no viable alternative. The majority of Cardano's web tooling is built with Javascript where JSON is the primary object representation format.
+
+Furthermore, even across non-Javascript based stacks, JSON enjoys wide tooling support, this improves the potential for builders to adopt this standard.
+
+### Bech32 for addresses
+
+We choose to use Bech32 as the representation for Cardano addresses. When compared to the alternative of base64, Bech32 gives the advantages of an included checksum and a human readable prefix. These add additional safety to the standard.
+
+## Path to Active
 
 ### Acceptance Criteria
 
+- [ ] One future ledger era schema is added
+- [ ] This standard is implemented within three separate tools, libraries, etc. 
```

Comment:
Why three? Is this from some more general CIP guideline?

Reply:
The acceptance criteria is meant to be a list of steps to make the CIP reach the `active` status
described [here](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#acceptance-criteria)